### PR TITLE
Refactor/favicon-update

### PR DESCRIPTION
**Pull request by petrKavulok**
### in this PR 
- favicon assets were updated with new logo
- asab-webui submodule update

Screenshot 2023-06-26 at 16 26 11
Screenshot 2023-06-26 at 16 26 53

### CODE REVIEW

File: public/media/teskalabs/img/favicon/favicon-template-395x395-cropped.png
Comment (Pe5h4) [inline public/media/teskalabs/img/favicon/favicon-template-395x395-cropped.png:1]: @petrKavulok Is this even used? I cant find the reference so I suppose this is obsolete and should be removed/correctly named

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@petrKavulok Looks good. Just one issue I have found. Please check

File: public/media/teskalabs/img/favicon/favicon-template-395x395-cropped.png
Comment (petrKavulok) [reply public/media/teskalabs/img/favicon/favicon-template-395x395-cropped.png:1]: ugh.. yes, sorry. This is the file I used as a template to create the other, smaller logos. It was included by accident

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +14,7 @@ To display last successful and unsuccessful login information, Seacat Auth servi
 
 ### Features
 
-- ASAB WebUI submodule version bump [[ae64c84](https://github.com/TeskaLabs/asab-webui/commit/ae64c84a78ecbbc32a1a02c5bf3bc196907b0263)] commit (INDIGO Sprint 230526, [!32](https://github.com/TeskaLabs/seacat-auth-webui/pull/32))
+- ASAB WebUI submodule version bump [[3a35738](https://github.com/TeskaLabs/asab-webui/commit/3a3573806e2964efbf5e5bb02c15acc9af41c8ac)] commit (INDIGO Sprint 230623, [!34](https://github.com/TeskaLabs/seacat-auth-webui/pull/34))
```
Comment (Pe5h4) [inline CHANGELOG.md]: @petrKavulok Dont update submodule in changelog in already released version. It has no use, instead put it to a release candidate :)

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**